currency_rate_check/models/account_move.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 VanMoof (<https://vanmoof.com>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, models
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    @api.multi
+    def _check_lock_date(self):
+        """Do not raise when the entry is prior to the lock date if 'bypass_update_checks'
+        is True in the context."""
+        if self._context.get('bypass_update_checks', False):
```

Comment:
this means any user can do an rpc call with this key to circumvent this check. Better do something like https://github.com/OCA/server-tools/commit/7d38fea57fc0c1ee9b81475f125d5e88107c5407